webapp/src/components/custom_post_types/post_type_recording.tsx

```diff
@@ -1,8 +1,73 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
+import {useSelector} from 'react-redux';
+import styled from 'styled-components';
+
+import {GlobalState} from '@mattermost/types/store';
+import IconAI from 'src/components/icons/ai';
+
+const aiPluginID = 'mattermost-ai';
+
+const useAIAvailable = () => {
+    //@ts-ignore plugins state is a thing
+    return useSelector<GlobalState, boolean>((state) => Boolean(state.plugins?.plugins?.[aiPluginID]));
+};
+
+type CallsPostButtonClickedFunc = (post: any) => void;
+
+const useCallsPostButtonClicked = () => {
+    return useSelector<GlobalState, CallsPostButtonClickedFunc>((state) => {
+        //@ts-ignore plugins state is a thing
+        return state['plugins-' + aiPluginID]?.callsPostButtonClicked;
```

Comment:
This is supplied by the AI plugin. That way all the logic for what to do when the button is clicked is handled on the AI plugin side.

Reply:
I like it, I think this is the right way of integrating between plugins.

Reply:
Same thing here as well. 
But otherwise, totally agree with Claudio -- this is the way to do it!